src/language/githubReleaseInstaller.ts

```diff
@@ -15,18 +15,29 @@ import { DafnyInstaller, getPreferredVersion } from './dafnyInstallation';
 import { configuredVersionToNumeric } from '../ui/dafnyIntegration';
 const ArchiveFileName = 'dafny.zip';
 
-function getDafnyPlatformSuffix(): string {
+function getDafnyPlatformSuffix(version: string): string {
+  const post312 = version.includes('nightly') || configuredVersionToNumeric(version) >= configuredVersionToNumeric('3.13');
   switch(os.type()) {
   case 'Windows_NT':
-    return 'win';
+    if(post312) {
```

Comment:
This seems like it would be cleaner if the `case` was inside a single `if` instead, since all three cases are different post 3.12

Reply:
Yeah, perhaps so.